### DISCUSSION TITLE
Call to similar albums always return 404

### DISCUSSION THREAD

**Discussion by kaste** [tidal-music/discussions#50] [Report bug]
I'm trying to get the similar albums but calling the URL `/albums/$id/similar` (e.g. https://api.tidal.com/v1/albums/72656630/similar?countryCode=DE&limit=100) always returns a 404.   The sample ID from the documentation `251380836` (iirc that's Beyoncé) works though.  

(Also the sibling request `"/artists/$id/similar"` works to get the list of similar artists.)

Is it just me and the countryCode?  What am I missing?

**Comment by stefanmoro** [comment]
Looks like you're calling the wrong endpoint (api.tidal.com is not available for 3rd party developers, only openapi.tidal.com). Please check the API reference: https://developer.tidal.com/api-reference?spec=catalogue&ref=get-similar-albums

**Reply by kaste** [reply]
We currently use api.tidal.com on an opensource media server.  I don't know how and when we would transition to the openapi endpoint as we need the actual flac urls from the endpoint.  However,  you noticed that the similar "Similar Artists" endpopint (`/artists/$id/similar`) works?
